README.md

```diff
@@ -25,11 +25,11 @@ export COMMIT="COMMIT-UUID"
 
 clang-tidy "<clang-tidy-configs>" | \
 ./codacy-clang-tidy-"<version>" | \
-curl -XPOST -L -H "project_token: $PROJECT_TOKEN"
+curl -XPOST -L -H "project_token: $PROJECT_TOKEN" \
     -H "Content-type: application/json" -d @- \
     "https://api.codacy.com/2.0/commit/$COMMIT/issuesRemoteResults"
 
-curl -XPOST -L -H 'project_token: $PROJECT_TOKEN' \
+curl -XPOST -L -H "project_token: $PROJECT_TOKEN"s \
```

Comment:
Is that s supposed to be there? Or is it a CMD+s gone wrong?

Reply:
It is a CMD+s gone wrong indeed 😆 
I think I might have to change keyboard...

Reply:
Fixed